tasks/managed/embargo-check/embargo-check.yaml

```diff
@@ -62,19 +71,56 @@ spec:
             exit 1
         fi
 
+        set +x # There are a lot of custom fields, don't want to pollute the logs
```

Comment:
I think more importantly you would leak the token, no? :)

Reply:
Completely forgot about that, nice call

Reply:
I am going to have to add more `set -x` stuff too around the actual curl call for this

Reply:
I didn't notice you already commented it, but I was referring to this `Also, both the AUTH_ARGS= and OUTPUT= would leak the token, so you'll need set +x here.`

Reply:
I moved the set command around a bit

Reply:
ok, looks good, thanks